circle.yml

```diff
@@ -1,17 +1,15 @@
 dependencies:
   override:
-    - make setup
+    - rm -rf node_modules
+    - case $CIRCLE_NODE_INDEX in 0) make setup ;; 1) nvm exec 0.12.7 make setup ;; 2) nvm exec 4 make setup ;; 3) nvm exec 5 make setup ;; 4) nvm exec 6 make setup ;; esac:
+        parallel: true
```

Comment:
Since parallelism is limited to 4x, let's drop one of the older Node versions. Having Node 0.10.x in the mix is useful, as it's a reasonable approximation of old browsers. CircleCI was able to flag sanctuary-js/sanctuary-type-classes#2 for its dependence on [`Array.of`][1], which is not defined in Node 0.10.x. Shall we drop Node 0.12.7?


[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/of